utils/sl/check_script.py

```diff
@@ -135,17 +136,20 @@ def write_variables(outfile, prefix, variables):
                 outfile.write("%s%s = None\n" % (prefix, variable))
 
         outfile.write("# pylint: enable=invalid-name\n")
+        outfile.write("# pylint: enable=import-outside-toplevel\n")
```

Comment:
You could probably use the on-line annotation, rather than enable/disable but I don't think it makes much difference.

Reply:
is that a comment at end of line?

Reply:
I think either end-of-line or on the previous line.  It doesn't matter though, this is fine.